### DISCUSSION TITLE
ImageGlass 9.0.11.502

### DISCUSSION THREAD

**Discussion by d2phap** [d2phap/ImageGlass#1897] [General]
<img width="640" alt="ImageGlass 9.0.11.502" src="https://raw.githubusercontent.com/ImageGlass/releases/main/screenshots/v9.0/9.0_b2.webp">

[![Downloads](https://img.shields.io/github/downloads/d2phap/imageglass/9.0.11.502/total?color=%232A7C91&label=downloads&style=for-the-badge)](https://imageglass.org/release/imageglass-9-0-11-502-52)

Please support this project by making a donation or purchasing the Microsoft Store version:
<a href="https://www.microsoft.com/store/productId/9N33VZK3C7TH?launch=true&mode=full&cid=GitHubRelease&referrer=appbadge&source=GithubD2phap">
  <img height="69" src="https://user-images.githubusercontent.com/3154213/197671083-4099d80f-07fc-4af9-b80d-6357f0968347.svg" />
</a>

## 🪁 System Requirements
- 🆕 Windows 10/11 64-bit, version 1809 (build 17763) or later
- [.NET Desktop Runtime 8 (64-bit)](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)
- [WebView2 Runtime 64-bit v119.0.2151 or later](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section)

### 🌱 New Features
- Added support for saving image as AVIF format
- Added support for custom WebView2 Runtime:
  + Create "WebView2_Runtime" folder in the [Configuration Folder](https://imageglass.org/docs/app-configs)
  + Download the [Fixed Version of WebView2 x64](https://developer.microsoft.com/en-us/microsoft-edge/webview2) (e.g. `Microsoft.WebView2.FixedVersionRuntime.124.0.2478.67.x64.cab`)
  + Renamed the downloaded file extension from `.cab` to `.zip` and extract it to a folder
  + Copy all files and folders where the `msedgewebview2.exe` file is located into the "WebView2_Runtime" folder
  + Restart ImageGlass
  + Open About dialog (Main menu > Help > About) to verify the WebView2 Runtime version
- Experiment feature (No UI yet):
  + Added support for opening the Save As dialog in the current image directory via setting `OpenSaveAsDialogInTheCurrentImageDir` (#1144)

### 🔮 Improvements
- Upgraded Magick.NET v13.7.0
- Upgraded libwebp v1.4.0
- Improved first launch time of ImageGlass
- Improved text message when WebView2 Runtime is not found and added a link to download (#1852)
- Added support for VarDCT Lossy for JPEG XL image format (#1864)
- Added support for selecting action buttons with arrow keys in dialog box (#1663)
- Added support for resetting all settings with Quick Setup dialog (#1694)

### 🐛 Bugfixes
- Fixed an issue where some PSD files could not be opened due to "incorrect check in the PSD coder" (#1806)
- Fixed an issue where some WEBP files could not be opened due to "insufficient image data" (#1807)
- Fixed an issue where some animated GIF files could not be saved as animated WEBP format (#1849)
- Fixed an issue where `igcmd.exe` still displayed UI without passing `--ui` param (#1835)
- Fixed an issue where ImageGlass could not work due to "An item with the same key has already been added" error (#1842, #1892)

**Full Changelog**: https://github.com/d2phap/ImageGlass/compare/8.12.4.30...9.0.11.502

<hr /><em>This discussion was created from the release <a href='https://github.com/d2phap/ImageGlass/releases/tag/9.0.11.502'>ImageGlass 9.0.11.502</a>.</em>

**Comment by luongson86** [comment]
Mình nâng cấp từ 9.0.10.201 lên 9.0.11.502 bằng cách tải file về cài đè. Sau khi cài xong thì ứng dụng không cách nào chạy lên được. Mình đã thử tải và cài lại bằng cách gỡ ứng dụng ra và cài mới hoàn toàn đều gặp tình trạng này. Buộc phải gỡ ra và cài lại bản 9.0.10.201.
-----------------------
Device name	LS-PC
Processor	AMD Ryzen 5 5600X 6-Core Processor                3.70 GHz
Installed RAM	32,0 GB
Device ID	44D62BCC-BFD6-44F0-82DE-818FD5055F96
Product ID	00330-80000-00000-AA931
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display
-----------------------
Bạn có gặp tình huống này bao giờ chưa? Cách xử lý như thế nào?

**Reply by d2phap** [reply]
@luongson86 Một số người gặp phải tình trạng này khi cài đặt InkScape với "System PATH":
https://github.com/d2phap/ImageGlass/issues/1935#issuecomment-2165762855

Giải pháp tạm thời là cài lại InkScape với tuỳ chọn "Do not add to PATH"

**Reply by luongson86** [reply]
Cám ơn bạn, hy vọng nó sẽ được khắc phục sớm trong bản cập nhập tiếp theo ❤️
